src/st/no0/41F98.c

```diff
@@ -108,6 +108,22 @@ void func_us_801C3308(Entity* self) {
     DestroyEntity(self);
 }
 
-INCLUDE_ASM("st/no0/nonmatchings/41F98", func_us_801C3334);
+void AxeKnightDeath() {
```

Comment:
This feels incorrectly named, looks more likely to be common death code across a few entities like axe knight and warg? But I've just copied this in here for now

Reply:
If it's this enemy it doesn't seem like the death explosions have much to with the wargs? 

https://www.youtube.com/watch?v=5HnhPNS0rqM&t=1007s

vs.

https://youtu.be/5HnhPNS0rqM?t=438

Looks like there's dedicated code for it in the entity function https://github.com/Xeeynamo/sotn-decomp/pull/1780/files#diff-18a47bb649a82f4e70725d56ace255bf4dd4d6a7a41e758bd3575b7c7d28a98aR357

Reply:
Was going off the fact this code is using `E_WARG_EXP_OPAQUE` and `entity->ext.wargpuff`, also based on where it sits in memory (pretty far from the other axe knight code which is all grouped up together) so my hypothesis was this was some kind of shared death code, but you may be right